### ISSUE TITLE
`Error: 'str' object has no attribute 'generate_usage'` when generating new config.

### ISSUE THREAD

**Issue by barmat** [wordfence/wordfence-cli#97]
The error occurs when running `wordfence configure` for a clean install.  After accepting the Cache directory defaults, the program exits.

```
Cache directory (default: ~/.cache/wordfence):
Error: 'str' object has no attribute 'generate_usage'
```

**Comment by barmat** [comment]
Should be fixed with ec4b7a6d6c28982313028d632a8223a41bf21c9e

**Comment by ewodrich** [comment]
Testing in conjunction with the original "Configure" case #72
